src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -64,11 +64,12 @@ public function getConfigTreeBuilder()
                         ->booleanNode('force_eager')->defaultTrue()->info('Force join on every relation. If disabled, it will only join relations having the EAGER fetch mode.')->end()
                     ->end()
                 ->end()
+                ->booleanNode('enable_doctrine_mongodb_odm')->defaultValue(false)->info('Enable Doctrine MongoDB ODM integration.')->end()
+                ->booleanNode('enable_doctrine_orm')->defaultValue(true)->info('Enable Doctrine ORM integration.')->end()
```

Comment:
Sorry, this would be the end yaml configuration. I'd put these two values in a sub `arrayNode` (`doctrine`) here:

```
->arrayNode('doctrine')
                    ->addDefaultsIfNotSet()
                    ->children()
```